src/devices/MotorHat/samples/README.md

```diff
@@ -0,0 +1,4 @@
+﻿# TODO: This needs to be determined
```

Comment:
please remove or paste relevant sections of the sample as an example and replace TODO